content/en/_index.md

```diff
@@ -234,6 +330,17 @@ ensure that it works.
         // Snippet content ...
     }
   {{< /tab >}}
+  {{< tab header="Node.js" >}}
+    function exampleSnippet(projectId, filePath) {
+        const result = `${projectId}/${filePath}`;
+
+        // Do this...
+        console.log('Result:', result);
+
+        // ...NOT this!
+        return result;
+    }
```

Comment:
Hmm this is interesting too - feels like we might want to be consistent with Java on this. @grayside thoughts?

Reply:
We also chatted about this one in the WG meeting, and the consensus was that we want to avoid showing the "wrong example" for now - there are some example of this in other readability guides, but they are able to show a red vs green example to more clearly mark which one was good and which one wasn't. 

So I think for now we only want the "right" way to do it showing.

Reply:
OK - done.